src/v35.js

```diff
@@ -26,7 +26,7 @@ export default function v35(name, version, hashfunc) {
       namespace = parse(namespace);
     }
 
-    if (namespace.length !== 16) {
+    if (namespace?.length !== 16) {
       throw TypeError('Namespace must be array-like (16 iterable integer values, 0-255)');
     }
```

Comment:
```suggestion
if (!Array.isArray(namespace) || namespace.length !== 16) {
      throw TypeError('namespace must be uuid string or an Array-like of 16 byte values');
    }```

Reply:
When I applied the suggested change you've made, there are total 8 failed test cases in `test/unit/v35.test.js`.
There are several test cases where the namespace parameter is given a string argument, while the above code checks if it is not an array then a TypeError will be throwed.

Reply:
Gah, sorry... I forgot `Array.isArray()` doesn't recognize typed arrays.  ("Wat???")

So, yeah, your original change (`?.`) should be fine.  Sorry for the confusion.